examples/flagging-bad-clients/docker-compose.yml

```diff
@@ -11,11 +11,23 @@ services:
       - "127.0.0.1:8889:8889"
     volumes:
       - ./hivemq/hivemq-config.xml:/opt/hivemq/conf/config.xml
-      - ./hivemq/hivemq.lic:/opt/hivemq/license/hivemq.lic
+      # - ./hivemq/hivemq.plic:/opt/hivemq/license/hivemq.plic
     healthcheck:
       test: [ "CMD", "curl", "-f", "http://localhost:8889/api/v1/health/readiness" ]
       interval: 5s
       retries: 10
+  activate_trial_mode:
```

Comment:
With this activation in place, the "policy-creator" probably needs a `dependsOn: activate_trial_mode`, so that trial mode is actually enabled before any policy is posted to HiveMQ. Else policies with redirect or user properties will be rejected when the timing is right.

Reply:
Oh that is good point, let me check how the relation can be expressed.